internal/services/v1/hash_test.go

```diff
@@ -419,11 +419,11 @@ func TestBulkCheckPermissionItemWithoutResourceIDHashStability(t *testing.T) {
 					},
 				},
 			},
-			"4fe0404757f60c7b",
+			"f518629690bd9dc0",
```

Comment:
these hashes should not be changing?

Reply:
May not be necessary to change

Reply:
The goal in this pull request was to switch everything to using the new method and types and implement the old method in terms of the new method. Changing the call hashes to use the new method keeps the code consistent.

Reply:
ok